apps/console/src/features/applications/components/settings/access-configuration.tsx

```diff
@@ -635,116 +699,291 @@ export const AccessConfiguration: FunctionComponent<AccessConfigurationPropsInte
-                                                : selectedProtocol
-                                        )
-                                            ? inboundProtocolConfig[
-                                                selectedProtocol === SupportedAuthProtocolTypes.OAUTH2_OIDC
+                { inboundProtocolList.length > 1
+                    ? ( inboundProtocolList.map(
+                        (protocol: string, index: number) => {
+                            return (
+                                Object.values(SupportedAuthProtocolTypes)
+                                    .includes(protocol as SupportedAuthProtocolTypes)
+                                    ? (
+                                        <AuthenticatorAccordion
+                                            key={ index }
+                                            globalActions={
+                                                !readOnly && [
+                                                    {
+                                                        icon: "trash alternate",
+                                                        onClick: handleProtocolDeleteOnClick,
+                                                        type: "icon"
+                                                    }
+                                                ]
+                                            }
+                                            authenticators={
+                                                [ {
+                                                    actions: [],
+                                                    content: protocol && (
+                                                        <InboundFormFactory
+                                                            onUpdate={ onUpdate }
+                                                            application={ application }
+                                                            isLoading={ isLoading }
+                                                            setIsLoading={ setIsLoading }
+                                                            certificate={ certificate }
+                                                            tenantDomain={ tenantName }
+                                                            allowedOrigins={ allowedOriginList }
+                                                            metadata={
+                                                                // There's no separate meta for `OAuth2/OIDC`
+                                                                // Apps. Need to use `OIDC` for noe.
+                                                                authProtocolMeta[
+                                                                    protocol ===
+                                                                                SupportedAuthProtocolTypes.OAUTH2_OIDC
```

Comment:
Shall we keep these in one line?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5161/commits/f26e73598f340f7ac2bf1196034aedf5ac209429